src/canmatrix/tests/test_frame_decoding.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import canmatrix.formats
+import os.path
+
+def loadDbc():
+    here = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I put this in global scope so it's accessible to all.  Since you accepted my PR with pathlib2 you can also `pathlib2.Path(__file__).resolve().parent`.  (i missed the resolve part in my use)  Below you just `here/'test_frame_decoding.dbc'`.

Reply:
feel free to change - I just did this simple solution when pathlib2 was not yet imported in project...

Reply:
deferred for later resolution.  https://github.com/ebroecker/canmatrix/issues/236#issuecomment-435590542